src/framework/theme/styles/_theming.scss

```diff
@@ -54,6 +54,35 @@ $nb-themes-export: () !global;
   @return $theme;
 }
 
+@function create-colors-palette($theme) {
+  $color-keys: (
+    color-primary: 20deg,
+    color-success:  20deg,
+    color-info: -10deg,
+    color-warning: 10deg,
+    color-danger: -20deg,
+  );
+
+  @each $key, $degree in $color-keys {
+    $color: map-get($theme, $key);
+
+    $color-300: tint($color, 14%);
```

Comment:
Shouldn't `tint`, `mix` etc. be removed as well? And I can't find usages of `create-colors-palette` function.

Reply:
this function allows generating colors from existing ones. We need to add it to the migration guide https://github.com/akveo/nebular/issues/1309.